src/models/userDao.js

```diff
@@ -119,11 +119,68 @@ const findUserByNickname = async (nickname) => {
   return user;
 };
 
+const findUserByIdWithOthers = async (userId) => {
```

Comment:
```suggestion
const findUserByIdWithHealthInfo = async (userId) => {
```
- Others라는 단어의 의미가 모호합니다.
- 다른 사람이라는 의미로 오인될 가능성이 있어, 보다 명확한 값으로 대체하시는 것을 추천드립니다

Reply:
수정했습니다!!!